scripts/debug_rom/Makefile

```diff
@@ -0,0 +1,31 @@
+# See LICENSE.SiFive for license details
+# Recursive make is bad, but in this case we're cross compiling which is a
+# pretty unusual use case.
+
+CC = $(RISCV)/bin/riscv64-unknown-elf-gcc
+OBJCOPY = $(RISCV)/bin/riscv64-unknown-elf-objcopy
```

Comment:
I don't think we do actually...

Anyway, the compiled contents *must* be XLEN-independent for the DebugModule to work (I did check that). This script is meant to be run once and then the code checked in, not as part of our flow.